neuralmonkey/tf_manager.py

```diff
@@ -1,7 +1,74 @@
-"""Interface between the data and TF session.
+"""==================
+TensorFlow Manager
+==================
 
-This module impelements the TensorFlow manager which encapsulates the graph
-execution in existing sessions.
+TensorFlow manager is a helper object in Neural Monkey which manages TensorFlow
+sessions, execution of the computation graph, and saving and restoring of model
+variables.
+
+This document describes TensorFlow Manager from the users' perspective: what
+can be configured in Neural Monkey with respect to TensorFlow.  The
+configuration of the TensorFlow manager is specified within the INI file in
+section with class ``tf_manager.TensorFlowManager``::
```

Comment:
To je zavádějící toholeto. Napsal, že že: both training and runing the model requires an object of class `tf_manager.TensorFlowManager` which needs to be specified in the configuration.